quixstreams/sinks/community/postgresql.py

```diff
@@ -238,20 +286,103 @@ def _add_new_columns(self, table_name: str, columns: dict[str, type]) -> None:
             with self._client.cursor() as cursor:
                 cursor.execute(query)
 
-    def _insert_rows(self, table_name: str, rows: list[dict]) -> None:
+    def get_current_primary_keys(self, table: str) -> list[str]:
+        if not self._client:
+            self.setup()
+        query = sql.SQL(
+            """
+            SELECT
+                kcu.column_name
+            FROM
+                information_schema.table_constraints tc
+            JOIN
+                information_schema.key_column_usage kcu
+                ON tc.constraint_name = kcu.constraint_name
+            WHERE
+                tc.constraint_type = 'PRIMARY KEY'
+                AND tc.table_name = {table}
+                AND tc.table_schema = {schema};
+            """
+        ).format(
+            schema=sql.Literal(self._schema_name),
+            table=sql.Literal(table),
+        )
+        with self._client.cursor() as cursor:
+            cursor.execute(query)
+            pks = list(sorted(row[0] for row in cursor.fetchall()))
+        return pks
+
+    def _add_new_primary_keys(self, table_name: str, pks: dict[str, type]) -> None:
+        current_pks = self.get_current_primary_keys(table_name)
+        _pks = list(sorted(pks.keys()))
```

Comment:
tiny redundancy, `sorted` returns a list
```suggestion
        _pks = sorted(pks.keys())
```